### PR TITLE
GPII-3590: reorganize and update docs

### PR DESCRIPTION
**Pull request by mrtyler**
I did some reorganization of the docs in preparation for sending them to new users.

I tried to make targeted commits, but big refactors like this can be hard to understand by looking at diffs -- especially for documentation.

You can review the rendered markdown here:
https://github.com/mrtyler/gpii-infra/tree/gpii-3590-docs

The big ideas here are:
* Make "Getting Started" path for new users clearer and more concise
* Extract some areas into separate documents linked from the main READMEs: one-time setup steps, how to contact ops, principles
* Copy relevant FAQ items from AWS to GCP

I suspect I'll find more things to change as I train new users on GCP, hence the WIP label. However I think there's enough here that an early look might be useful for reviewers.

### CODE REVIEW

File: common/README.md [hunk 1]
```diff
@@ -73,13 +73,14 @@ From the root of the repository:
...
 
-The projects are defined in the directory tree `common/live/stg/infra/`. To add a new user make a copy of the user `john`, edit the file john/terraform.tfvars and change the `project_name` variable. Example:
+The projects are defined in the directory tree `common/live/stg/infra/`. To add a new user make a copy of the user `john`, edit the file john/terraform.tfvars and change the `project_*` variables. Example:
 
 ```
 cd common/live/stg/infra/
 cp -r john $USER
 #(edit $USER/terraform.tfvars)
   project_name = "dev-$USER"
+  project_owner = "$USER@RtF"
```
Comment (stepanstipl) [inline common/README.md]: Worth to include the `user:` prefix here? ;)

Comment (mrtyler) [reply common/README.md]: Yes :)

**Comment by natarajaya** [comment]
LGTM!

**Comment by mrtyler** [comment]
I thought I had put the work-in-progress label on this PR but now I can't find it.

Anyway, I probably should have merged this earlier, but I did the initial GCP training today and added some things we discovered during that process. I think this is ready for a last review before merging, if anyone is so inclined.

I added a parking lot item about running `update_exekube`.

**Comment by stepanstipl** [comment]
> I thought I had put the work-in-progress label on this PR but now I can't find it.

:) screenshot 2019-02-20 at 12 43 14

LGTM

**Pull request merged**